homeassistant/components/fully_kiosk/button.py

```diff
@@ -0,0 +1,93 @@
+"""Fully Kiosk Browser button."""
+from __future__ import annotations
+
+from collections.abc import Callable
+from dataclasses import dataclass
+
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import FullyKioskDataUpdateCoordinator
+from .entity import FullyKioskEntity
+
+
+@dataclass
+class FullyButtonEntityDescriptionMixin:
+    """Mixin to describe a Fully Kiosk Browser button entity."""
+
+    press_action: Callable
```

Comment:
```suggestion
    press_action: Callable[[FullyKiosk], Awaitable[None]]
```

Reply:
This change is failing mypy:
`error: Returning Any from function declared to return "Awaitable[None]"  [no-any-return]`

Reply:
Right the upstream package isn't typed.
Will push a small change for it.